CloudSQLReplicator/google/cloud/pontem/sql/replicator/util/gcp_api_util.py

```diff
@@ -96,6 +103,116 @@ def create_cloudsql_instance(database_instance_body=None,
 
   return response
 
+def create_cloudsql_source_representation(
+      ip_address,
+      port,
+      db_version=DEFAULT_2ND_GEN_DB_VERSION,
+      region=DEFAULT_2ND_GEN_REGION,
+      source_representation_name=None,
+      source_representation_body=None,
+      project=None,
+      credentials=None):
+  """Creates a source representation of an external master.
+
+
+  If source representation is included source_representation_name, ip_address,
+  port, db_version and region are ignored.
+
+  Args:
+    source_representation_name (str): The instance name of the external master.
+    ip_address (str): The ip address of the external master.
+    port (str): Port that will be used for replication.
+    region (str): Region source representation will be created.
+    database_version (str): MySQL database version
+    source_representation_body (JSON): Creation options for source
+      representation.
+    project (str): Project ID where replica will be created.
+      credentials (google.auth.Credentials): Credentials to authorize client.
```

Comment:
should the URL be indented?

Reply:
Did not mean to leave the URL in the docstring.